File: Backend/src/utils/apiResponse.js
class ApiResponse {
  static success(message , data , statusCode) {
    return {
      message,
      data,
      statusCode
    };
  }

  static error(message , statusCode) {
    return {
      message,
      statusCode,
    };
  }
}

module.exports = ApiResponse;

File: BackendFastApi/api/utils/apiResponse.py
from typing import Any, Optional

class ApiResponse:
    @staticmethod
    def success(message: str, data: Any = None, status_code: int = 200):
        return {
            "message": message,
            "data": data,
            "statusCode": status_code
        }
    
    @staticmethod
    def error(message: str, status_code: int = 500):
        return {
            "message": message,
            "statusCode": status_code
        }
